lib/foreman_templates.rb

```diff
@@ -1,4 +1,8 @@
 require 'foreman_templates/engine'
 
 module ForemanTemplates
+  BASE_SETTING_NAMES = %w(repo branch dirname filter negate)
+  COMMON_SETTING_NAMES = BASE_SETTING_NAMES | %w(verbose)
```

Comment:
Where is this used?

Reply:
nowhere, I just felt to have the common_settings full list. but I can do without.